DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -25,6 +25,21 @@ public SectionContentService(IDbConnection connection, ILogger<SectionContentSer
 
         public SectionContent? GetSectionContent(int customisationId, int candidateId, int sectionId)
         {
+            // NextSectionID is the ID of the next section in the course, according to SectionNumber
+            // or null if the last in the course.
+
+            // Find these by making a list of other sections in the course. A section is valid if it is not archived
+            // and has one of the following:
+
+            // A tutorial can be viewed; if it has CustomisationTutorials.Status 1.
```

Comment:
Should be "A tutorial which can be viewed"? 🙂

Reply:
Yep, sorry! Fixed in c6e65d9 🙂